packages/analytics/package.json

```diff
@@ -15,7 +15,7 @@
         "lint:js": "yarn g:eslint '**/*.{ts,tsx,js}'",
         "test:unit": "yarn g:jest -c ../../jest.config.base.js",
         "type-check": "yarn g:tsc --build",
-        "build:lib": "yarn g:rimraf lib && yarn g:tsc --build ./tsconfig.lib.json"
+        "build:lib": "yarn g:rimraf lib && yarn g:tsc --build tsconfig.lib.json && ../../scripts/replace-imports.sh ./lib"
```

Comment:
Imports are replaced right after build now to be more consistent. I will probably create some universal script to build libs because this starting to be too long.